geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/z2/Z2Index.scala

```diff
@@ -30,6 +30,10 @@ case object Z2Index extends AccumuloFeatureIndexType with Z2WritableIndex with Z
   // step needed (due to the mask) to bump up the z value for a complex geom
   val GEOM_Z_STEP: Long = 1L << (64 - 8 * GEOM_Z_NUM_BYTES)
 
+  // in 1.2.5/6, we stored the CQ as the number of rows, which was always one
+  // this wasn't captured correctly in a versioned index class, so we need to delete both possible CQs
+  private val DeleteText = new Text("1,")
```

Comment:
it was 'number of rows' comma 'normal col qualifier' - number of rows was always 1 (otherwise it used the xz index), and normal col qualifier was always empty